### PR TITLE
Correcting the assign scalar field for components in process

### PR DESCRIPTION
**Pull request by loumalouomega**

### CODE REVIEW

**Comment by loumalouomega** [comment]
The component variable was missing, so I added it

**Review by RiccardoRossi — COMMENTED** [review]
Hi Vicente, what is the change you did in this?
i mean, apart for cosmetic/naming changes

**Comment by loumalouomega** [comment]
@RiccardoRossi I added the component variable (for example LINE_LOAD_X) because until now it was possible to use just with pure double variables

File: kratos/processes/assign_scalar_field_to_conditions_process.h [hunk 1]
```diff
@@ -105,18 +109,22 @@ class AssignScalarFieldToConditionsProcess : public Process
...
         const double& rCurrentTime = rCurrentProcessInfo[TIME];
 
 
-        if( KratosComponents< Variable<Vector> >::Has( mvariable_name ) ) //case of double variable
+        if( KratosComponents< Variable<Vector> >::Has( mVariableName ) ) //case of double variable
         {
-            InternalAssignValue<>(KratosComponents< Variable<Vector> >::Get(mvariable_name), rCurrentTime);
+            InternalAssignValue<>(KratosComponents< Variable<Vector> >::Get(mVariableName), rCurrentTime);
+        }
+        else if( KratosComponents< array_1d_component_type >::Has( mVariableName ) ) //case of component variable
```
Comment (loumalouomega) [inline kratos/processes/assign_scalar_field_to_conditions_process.h:121]: @RiccardoRossi This is what I meant

**Comment by RiccardoRossi** [comment]
@loumalouomega , ok, so just add the corresponding test case to the test_processes.py and i'll accept the commit

cheers
Riccardo

**Comment by loumalouomega** [comment]
OK

**Comment by loumalouomega** [comment]
OK, done

**Comment by loumalouomega** [comment]
@RiccardoRossi This is done

**Review by rubenzorrilla — APPROVED** [review]
Ok! Thanks @loumalouomega  and sorry for the late review.

**Comment by loumalouomega** [comment]
No worries

**Pull request merged**